ci/ray_ci/container.py

```diff
@@ -6,8 +6,11 @@
 
 import ci.ray_ci.bazel_sharding as bazel_sharding
 
-DOCKER_ECR = "029272617770.dkr.ecr.us-west-2.amazonaws.com"
-DOCKER_REPO = "rayproject/citemp"
+_DOCKER_ECR_REPO = os.environ.get(
+    "RAYCI_WORK_REPO",
+    "029272617770.dkr.ecr.us-west-2.amazonaws.com/rayproject/citemp",
+)
+_RAYCI_BUILD_ID = os.environ.get("RAYCI_BUILD_ID", "unknown")
```

Comment:
I tried to throw an exception when `RAYCI_BUILD_ID` is empty, but then would break several tests